hard_constraint:
    lower:
        - -225.8020
        - -1.9722
        - -2.6878
        - -2.1293
        - 0.0
        - 0.0
        - 0.0
        - 0.0
        - 0.0
    upper:
        - 225.8020
        - 1.3090
        - 2.6878
        - 2.1293
        - 0.0
        - 0.0
        - 0.0
        - 0.0
        - 0.0

#END